Source/WebKit/Platform/IPC/cocoa/DaemonConnectionCocoa.mm

```diff
@@ -77,6 +77,7 @@
 #endif
         }
         if (event == XPC_ERROR_CONNECTION_INTERRUPTED) {
+            WTFLogAlways("Connetion to mach service %s is interrupted", weakThis->m_machServiceName.data());
```

Comment:
Should we use RELEASE_LOG with a WebKit log category?

Reply:
Sure; I used `WTFLogAlways` to match the log lines above; but RELEASE_LOG works too. Will update.
Thanks for review!